### ISSUE TITLE
Reverse a profile's list of entries

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network-pulse#934]
Currently, a profile lists related creator entries from oldest to newest. Let's switch to put the newest first. 

Not urgent, but good to land as we have more activity from fellows and global sprint. See [example](https://www.mozillapulse.org/profile/209).

**Comment by mmmavis** [comment]
@Pomax sounds like a API side of task to me. True?

@xmatthewx do we want both `published_entries` and `created_entries` to order from newest to oldest?

**Comment by xmatthewx** [comment]
@mmmavis it looks like the api allows you to specify reverse ordering with `-` like `?ordering=-created`

**Comment by mmmavis** [comment]
@xmatthewx we get `published_entries` and `created_entries` from the `/profiles` endpoint, not the `/entries` endpoint. (e.g., https://api.mozillapulse.org/api/pulse/profiles/70/) I just tried it with `?ordering=-created` but it didn't seem to change anything.

**Comment by xmatthewx** [comment]
Ah, I see. I opened an API issue https://github.com/mozilla/network-pulse-api/issues/324

**Comment by xmatthewx** [comment]
I believe this is now unblocked by https://github.com/mozilla/network-pulse-api/pull/326

**Comment by mmmavis** [comment]
@alanmoo it's unclear to me what param I should pass to get `published_entries` and `created_entries` both sorted from newest to oldest projects. Is https://github.com/mozilla/network-pulse-api/pull/326 a fix for the Fellowship directory instead and not for this ticket?

**Comment by xmatthewx** [comment]
I was mistaken @mmmavis. Oops. You're right, that fix was ordering by name for directory, not ordering of entries on profiles. You're still blocked. And this is lower priority than the directory.
